packages/vue/src/components/molecules/SfSelect/SfSelect.vue

```diff
@@ -95,10 +102,10 @@ export default {
       default: "",
     },
   },
-  data() {
```

Comment:
There's an error in the console about mutating a prop directly so maybe data shouldn't be deleted, wdyt @AdamPawlinski ? :)

Reply:
It is connected with storybook, so it is on the dev side to use it correctly. Of course we should remove it finally, but now I consider it less important. I tried to add additional computed property for that, but it caused other errors, so I left it like this for now.